core/domain/question_services.py

```diff
@@ -217,3 +217,11 @@ def update_question(
     updated_question = apply_change_list(question_id, change_list)
     _save_question(
         committer_id, updated_question, change_list, commit_message)
+
+def get_new_question_id():
```

Comment:
In the tests below, have the question id be taken from this function, so you can test this function also.

Reply:
The question id generated is a random hash of 16 chars, so I don't think it could be tested.

Reply:
No, I meant to generate a random question id, and use that to create the sample questions.
Also, lint tests seem to be failing.